docs/tutorial.rst

```diff
@@ -210,7 +210,6 @@ this type is imposed based on the variable name. The current rules are pretty
 simple:
 
 * ``\w*PATH``: any variable whose name ends in PATH is a list of strings.
-* ``\w*DIRS``: any variable whose name ends in DIRS is a list of strings.
```

Comment:
Why remove this from the docs

Reply:
DIRS is very similar to PATH

Reply:
Because there is no DIRS variable in the code. I'm glad that you ask about it because may be it was in the code and deleted now in issues reason? If no I can add DIRS regex like a PATH.

Reply:
Oh really? That seems like a bug potentially

Reply:
`\w*DIRS` added with test.